pylocus/lateration.py

```diff
@@ -30,21 +30,24 @@ def get_lateration_parameters(real_points, indices, index, edm, W=None):
     return anchors, w, r2
 
 
-def SRLS(anchors, W, r2, print_out=False):
+def SRLS(anchors, W, r2, rescale=False, print_out=False):
     '''Squared range least squares (SRLS)
 
     Algorithm written by A.Beck, P.Stoica in "Approximate and Exact solutions of Source Localization Problems".
 
     :param anchors: anchor points
+    :param W: weights for the measurements
     :param r2: squared distances from anchors to point x.
+    :param rescale: Optional paramters. This deals with squared range measurements whose global scale has been lost.
```

Comment:
what do the "rescale" parameters represent? Without taking a piece of paper and writing out the formulas it's not obvious to me. Could this be explained in one simple sentence?